Source/Fuse.Triggers/Trigger.uno

```diff
@@ -341,8 +341,11 @@ namespace Fuse.Triggers
 		{
 			if (!_isStarted)
 			{
-				if (!IsRootingStarted)
-					throw new Exception( "Trigger started prior to being rooted: " + this );
+				if (!IsRootingStarted){
```

Comment:
Code-style: `{` that should've been on its own line.

Reply:
sorted